pkgs/development/python-modules/insightface/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, albumentations
+, buildPythonPackage
+, cython
+, easydict
+, fetchPypi
+, matplotlib
+, mxnet
+, numpy
+, onnx
+, opencv4
+, prettytable
+, pythonOlder
+, scikit-image
+, scikit-learn
+, tensorboard
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "insightface";
+  version = "0.7.3";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-8ZH3GWEuuzcBj0GTaBRQBUTND4bm/NZ2wCPzVMZo3fc=";
+  };
+
+  nativeBuildInputs = [
+    cython
+  ];
+
+  propagatedBuildInputs = [
+    easydict
+    matplotlib
+    mxnet
+    numpy
+    onnx
+    opencv4
+    scikit-learn
+    scikit-image
+    tensorboard
+    tqdm
+    albumentations
+    prettytable
+  ];
+
+  doCheck = false; # Upstream has no tests
```

In that case it would be good to have `pythonImportsCheck`. The following imports are mentioned in their [README](https://github.com/deepinsight/insightface/tree/master/python-package).

```suggestion
	pythonImportsCheck = [
	  "insightface"
	  "insightface.app"
	  "insightface.data"
	];

  doCheck = false; # Upstream has no tests
```

---

Done!